docs/changes/3.0.0.rst

```diff
@@ -7,3 +7,15 @@ Contributors to this Release
 Many thanks to all of our talented and stylish contributors for this release!
 
 - `screamingskulls <https://github.com/screamingskulls>`_
+
+Features
```

Comment:
It looks like you happened to commit the `towncrier` output.

Reply:
oh snap

Reply:
fixed